vendor/vendor.json

```diff
@@ -74,6 +74,12 @@
 			"revision": "97fb8dcd4a63b099605dc2f4dbe688e56de3bd00",
 			"revisionTime": "2019-01-30T09:05:45Z"
 		},
+		{
+			"checksumSHA1": "QZGRiUD71eXUoEre9NoV/Efz2Xk=",
+			"path": "github.com/aliyun/alibaba-cloud-sdk-go/services/cas",
+			"revision": "20c7e19e627e4c58c522936a103a2d7f266bc8c8",
+			"revisionTime": "2019-02-21T02:24:13Z"
+		},
 		{
 			"checksumSHA1": "AX+tkIg3vXRmDiyiF99V11jAegM=",
 			"path": "github.com/aliyun/alibaba-cloud-sdk-go/services/cbn",
```

Comment:
Please add docs for the datasource and resource.

Reply:
done